scheduler-server/build.gradle

```diff
@@ -9,8 +10,48 @@ java {
   }
 }
 
+node {
+  nodeProjectDir = file("${project.projectDir.toPath().resolve("scheduling-dsl-compiler")}")
+  version = '16.4.0'
+  npmVersion = '8.5.0'
+  download = true
+}
+
+task assembleSchedulingDSLCompiler(type: NpmTask) {
```

Comment:
Minor naming note – I usually prefer to follow camel case for abbreviations/acronyms. Microsoft has a little doc. about this: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/141e06ef(v=vs.71)?redirectedfrom=MSDN

Basically this helps avoid situations where you have `parse DBM XML` written as `parseDBMXML` by using `parseDbmXml`. Still a bit ugly but personally I find it easier to read/understand.

Reply:
I'm guessing you'll say the same thing about `SchedulingDSL` and `SchedulingDSLCompilationService` 😝

Reply:
Lol yup, no obligation to change this though. Pros/cons to both conventions.